.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: olafurpg/setup-scala@v10
-      - run: sbt docs/mdoc
+      - run: sbt '++2.12.13 docs/mdoc'
```

Comment:
Isn't this the default version?

Reply:
It no longer is - I had to set 2.13 to be the default for the "Scala 3 module depending on 2.13 module to work"

Here's the failure: https://github.com/scalameta/mdoc/runs/2331363147#step:4:256

`cli` module is a dependency of `mdoc`, which is a dependency of `docs` and it needs to be on 2.12 for I guess reusing the SBT plugin code

Reply:
Is this still necessary after #493? Evilplot has 2.13 releases on Sonatype